packages/core/src/fhir-to-cda/cda-generators.ts

```diff
@@ -0,0 +1,27 @@
+import { Bundle, Patient, Organization } from "@medplum/fhirtypes";
+import { constructRecordTargetFromFhirPatient } from "./cda-templates/record-target";
+import { constructAuthor } from "./cda-templates/author";
+import { constructCustodian } from "./cda-templates/custodian";
+import { constructStructuredBody } from "./cda-templates/structured-body";
+import { constructClinicalDocumentXML } from "./cda-templates/clinical-document";
+
+export function generateCdaFromFhirBundle(fhirBundle: Bundle): string {
+  const patientResource = fhirBundle.entry?.find(
+    entry => entry.resource?.resourceType === "Patient"
+  )?.resource as Patient;
+  const organizationResources = fhirBundle.entry?.find(
+    entry => entry.resource?.resourceType === "Organization"
+  )?.resource as Organization;
+
+  const recordTarget = constructRecordTargetFromFhirPatient(patientResource);
+  const author = constructAuthor(organizationResources);
```

Comment:
This has been marked as resolved but I still see we're using the Org and no comment explaining why

Reply:
Discussed [here](https://metriport.slack.com/archives/C04DMKE9DME/p1713460644411499?thread_ts=1713448810.552229&cid=C04DMKE9DME), can't use doc ref's `author` from FHIR since we're building the CDA off of resources.